packages/apalis-sql/src/lib.rs

```diff
@@ -358,5 +358,40 @@ macro_rules! sql_storage_tests {
                 assert_eq!(ctx.priority(), prio);
             }
         }
+
```

Comment:
One more thing, could you add a test that actually runs the job? Something like this:
```rs
#[tokio::test]
async fn integration_test_shedule_and_run_job() {
    let current = Utc::now();
    let schedule_time = Utc::now() + Duration::from_secs(60);
    let mut storage = setup_test_wrapper().await;
    storage
        .schedule(email_service::example_good_email(), schedule_time)
        .await
        .unwrap();

    let (job_id, res) = storage.execute_next().await.unwrap();
    assert_eq!(res, Ok("()".to_owned()));
    apalis_core::sleep(Duration::from_secs(1)).await;
    let job = storage.fetch_by_id(&job_id).await.unwrap().unwrap();
    let ctx = job.parts.context;
    assert_eq!(*ctx.status(), State::Done);
    assert!(ctx.lock_at()>  schedule_time));
}
```